app/models/user.rb

```diff
@@ -4,6 +4,7 @@ class User < ApplicationRecord
   before_save :email_downcase
   before_create :create_activation_digest
 
+  has_many :microposts, dependent: :destroy
```

Comment:
tại vì dịch sang tiếng việt thấy hơi sai sai thế mình xin viết tiếng anh nhé :)
dependent: :destroy arranges for the dependent microposts to be destroyed when the user itself is destroyed. prevents userless microposts from being stranded in the database when admins choose to remove users from the system.